src/main/java/com/juniorstart/juniorstart/payload/EmploymentHistoryRequest.java

```diff
@@ -0,0 +1,25 @@
+package com.juniorstart.juniorstart.payload;
+
+import com.juniorstart.juniorstart.model.UserProfile;
+import jakarta.validation.constraints.NotBlank;
+import lombok.Builder;
+import lombok.Data;
+
+@Data
+@Builder
+public class EmploymentHistoryRequest {
+
```

Comment:
Chyba request powinien mieć wszystko w validacji ale to nie wiem co w tym requescie Ci jest potrzebne

Reply:
Ups, to jest pozostałość , usuwam.